service/controller/resource/prometheusautoscaler/resource.go

```diff
@@ -0,0 +1,106 @@
+package prometheusautoscaler
```

Comment:
This is unrelated to this PR but should we not move all prometheus related resources under prometheus/ so we could have resources/prometheus/prometheus and resources/prometheus/ingress resources/prometheus/autoscaling ?

Reply:
I don't see how this is more related to prometheus than other resources?

Also I was hopping to see something named `verticalpodautoscaler`.

Reply:
As this is not prometheus specific, sure :)

Reply:
Sure, initially I thought of it as related to Prometheus, but yeah I see, it just creates a VPA resource, it just happens to be for Prometheus. I'll rename it.

Reply:
Done.